simulator/mips/mips_instr.cpp

```diff
@@ -481,6 +481,14 @@ void MIPSInstr::set_v_dst( uint32 value)
             case 4: v_dst = value; break;
             default: assert( false);
         }
+        if(operation == OUT_I_LOADR)
+            operation = OUT_I_LOAD;
+    }
+    else if ( operation == OUT_I_LOADR)
+    {
+        value &= 0x00FFFFFF;
+        value += v_imm << 24;
```

Comment:
I am not sure that you may generate mask using only immediate value, as address base may be unaligned as well.

Reply:
How address base can be unaligned?

Reply:
See example below